### PR TITLE
Add colorsEqualEpsilon to ModelSpec test

### PR DESCRIPTION
**Pull request by OmarShehata**
This fixes the test failure in https://github.com/AnalyticalGraphicsInc/cesium/issues/7073. 

I figured it would be useful to be able to test if colors are close enough so our tests are more flexible. Feedback is welcome!

I don't think the color change is due to https://github.com/AnalyticalGraphicsInc/cesium/pull/6877 because it looks like the `scene` in the spec doesn't have a globe initialized, so I assume that means it's not rendering the atmosphere. The box itself still looks fine in CesiumJS.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/relax-test/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hpinkos** [comment]
@OmarShehata you should be able to do `expect(color).toEqualEpsilon(value, epsilon)`

**Comment by hpinkos** [comment]
We have a custom jasmine matcher that should automatically call [Color.equalsEpsilon](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/Color.js#L564-L571) when you do `expect().toEqualEpsilon`

**Comment by OmarShehata** [comment]
Ah, didn't realize that already existed. Thanks for pointing that out @hpinkos ! This seems to work.

**Comment by hpinkos** [comment]
Thanks @OmarShehata!  The tests pass locally now.  There was a weird test failure in the travis build though.  I just restarted it to make sure it wasn't related to this change.  I'll merge this if that passes

**Comment by hpinkos** [comment]
@OmarShehata I'm not sure exactly why, but the change you made is causing all of the model tests to fail in travis.  I'm guessing something isn't being cleaned up correctly, so after your tests runs it's causing `scene.frameState.commandList` to have an extra command.

You can reproduce this locally by running `npm run test -- --browsers FirefoxHeadless --webgl-stub --failTaskOnError --suppressPassed` on the command line

File: Specs/Scene/ModelSpec.js [hunk 1]
```diff
@@ -945,8 +945,11 @@ defineSuite([
         return loadModel(boxGltf2Url).then(function(m) {
             verifyRender(m);
             m.show = true;
-            expect(scene).toRender([169, 3, 3, 255]); // Red
-            primitives.remove(m);
+
+            expect(scene).toRenderAndCall(function(rgba) {
+                expect(rgba).toEqualEpsilon([169, 3, 3, 255], 5); // Red
+                primitives.remove(m);
```
Comment (hpinkos) [inline Specs/Scene/ModelSpec.js]: Maybe this needs to be moved outside the function block?

**Comment by OmarShehata** [comment]
I think you were right @hpinkos !

**Comment by hpinkos** [comment]
Works great @OmarShehata, thanks!

**Pull request merged**